tests/gnrc_tcp/tests/05-garbage-pkts.py

```diff
@@ -97,17 +97,16 @@ def test_short_header(child, src_if, src_ll, dst_if, dst_l2, dst_ll, dst_port):
 @testfunc
 def test_send_ack_instead_of_syn(child, src_if, src_ll,
                                  dst_if, dst_l2, dst_ll, dst_port):
-    # check if entering _event_loop in `@testfunc` decorator generates debug
-    # output
-    debug = child.expect([pexpect.TIMEOUT, 'GNRC_TCP'], timeout=1)
-    # with debug output the node can be overwhelmed by packets + output
+    # check if any debug output is generated during `@testfunc` decorator 
+    # execution. If so, send fewer packets to prevent the target node
+    # from being overwhelmed by packets and output.
+    debug = child.expect([pexpect.TIMEOUT,
+                          r'GNRC_TCP: Enter "\S+", File: .+(\d+)\s"'], timeout=1)
```

Comment:
Wait... there is an error in here:
```suggestion
                          r'GNRC_TCP: Enter "\S+", File: .+(\d+)\s'], timeout=1)
```
the `\s` is supposed to match the line end and AFAIK there is no `"` at the beginning of a line ;-)

Reply:
Hint: If you just click on "Commit suggestion" that change is just added as a commit ;-).

Reply:
What was not an issue?

Reply:
I'll changed it anyway. If the current state is fine by you I'll squash now.

Reply:
For me currently the new regex doesn't work either. I'm currently debugging.